app/code/Magento/Ui/view/base/web/js/form/element/image-uploader.js

```diff
@@ -80,8 +80,11 @@ define([
             if (this.mediaGallery.initialOpenSubpath) {
                 openDialogUrl += '&current_tree_path=' + Base64.mageEncode(this.mediaGallery.initialOpenSubpath);
             }
+            this.mediaGallery.options ?
+                  this.mediaGallery.options.targetElementId = $buttonEl.attr('id') :
+                  this.mediaGallery.options = false;
```

Comment:
I don't think it's a good practice to have different types in the same variable, in this case you could have in `this.mediaGallery.options` either an object either boolean